pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/TenantsBase.java

```diff
@@ -120,58 +124,34 @@ public void getTenantAdmin(@Suspended final AsyncResponse asyncResponse,
     public void createTenant(@Suspended final AsyncResponse asyncResponse,
             @ApiParam(value = "The tenant name") @PathParam("tenant") String tenant,
             @ApiParam(value = "TenantInfo") TenantInfoImpl tenantInfo) {
-
-        final String clientAppId = clientAppId();
-        try {
-            validateSuperUserAccess();
-            validatePoliciesReadOnlyAccess();
-            validateClusters(tenantInfo);
-            NamedEntity.checkName(tenant);
-        } catch (IllegalArgumentException e) {
-            log.warn("[{}] Failed to create tenant with invalid name {}", clientAppId(), tenant, e);
-            asyncResponse.resume(new RestException(Status.PRECONDITION_FAILED, "Tenant name is not valid"));
```

Comment:
It seems that we have removed this exception. It will make users confused if will just throw the `IllegalArgumentException` without other information.

Reply:
Good catch! fixed.